lib/constants/fieldSearchConfigurations.js

```diff
@@ -73,6 +73,12 @@ export const fieldSearchConfigurations = {
     startsWith: 'administrativeNotes all "%{query.query}*"',
     containsAny: 'administrativeNotes any "%{query.query}"',
   },
+  placeOfPublication: {
+    exactPhrase: 'publication.place=="%{query.query}"',
+    containsAll: 'publication.place=="%{query.query}"',
+    startsWith: 'publication.place=="%{query.query}*"',
+    containsAny: 'publication.place=="%{query.query}"',
```

Comment:
The `exactPhrase`, `containsAll` and `containsAny` are the same, is this correct?

Reply:
Yes, it's a term field so only `==` operator is supported